js-apps/meep-frontend/src/js/containers/exec/exec-page-container.js

```diff
@@ -46,6 +47,7 @@ import { execChangeScenarioList} from '../../state/exec';
 import {
   uiChangeCurrentDialog,
   uiExecChangeEventCreationMode,
+  uiExecChangeViewMode,
   uiExecChangeEventAutomationMode,
   uiExecChangeEventReplayMode,
   uiExecChangeDashCfgMode,
```

Comment:
Why create a new ViewMode? Can you reuse the DashCfgMode instead?

Recommend keeping DashCfg state to manage Dashboard configuration (view configuration) pane.
Perhaps rename view-pane to dashboard-cfg-pane or dashboard-pane.

Reply:
removed viewMode.. now only reusing DashCfgMode. Rename view pane to DashCfgPane.